src/scenes/PlantingSitesRouter/view/PlantingSitesList.tsx

```diff
@@ -19,33 +19,7 @@ import PlantingSiteTypeSelect from 'src/scenes/PlantingSitesRouter/edit/Planting
 import { useOrganization, useTimeZones } from 'src/providers/hooks';
 import { setTimeZone, useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
 import useForm from 'src/utils/useForm';
-
-type SiteProperty = keyof PlantingSiteSearchResult;
-
-const getVal = (site: PlantingSiteSearchResult, key: SiteProperty) => {
-  if (key === 'numPlantingSubzones') {
-    return site['numPlantingSubzones(raw)'] ?? 0;
```

Comment:
One thing to note, the `sortResults` function I made doesn't fall back to 0, is that OK here? Will the search API return a result without this key? Easy to modify if we should always fall back to 0.

Reply:
In this case the BE has undefined values and we want to fallback to 0. Thanks for checking 🙏